<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PartsTracker</class>
 <widget class="QMainWindow" name="PartsTracker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1237</width>
    <height>908</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Parts Tracker</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>0</y>
      <width>1241</width>
      <height>881</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QTabWidget" name="tab_widget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="tabPosition">
        <enum>QTabWidget::North</enum>
       </property>
       <property name="tabShape">
        <enum>QTabWidget::Triangular</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="asssembly_listing">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <attribute name="title">
         <string>Assembly List</string>
        </attribute>
        <widget class="QFrame" name="frame">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>-1</y>
           <width>241</width>
           <height>51</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <widget class="QPushButton" name="button_expand_tree">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>10</y>
            <width>115</width>
            <height>36</height>
           </rect>
          </property>
          <property name="text">
           <string>Expand Tree</string>
          </property>
         </widget>
         <widget class="QPushButton" name="button_collapse_tree">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>10</y>
            <width>115</width>
            <height>36</height>
           </rect>
          </property>
          <property name="text">
           <string>Collapse Tree</string>
          </property>
         </widget>
        </widget>
        <widget class="QTreeWidget" name="assembly_tree_widget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>60</y>
           <width>1222</width>
           <height>817</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOn</enum>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
         </property>
         <property name="columnCount">
          <number>8</number>
         </property>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string notr="true">2</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string notr="true">3</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string notr="true">4</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string notr="true">5</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string notr="true">6</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string notr="true">7</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string notr="true">8</string>
          </property>
         </column>
        </widget>
       </widget>
       <widget class="QWidget" name="parts_list">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <attribute name="title">
         <string>Parts List</string>
        </attribute>
        <widget class="QTableWidget" name="parts_table_widget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>1222</width>
           <height>817</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <property name="gridStyle">
          <enum>Qt::DotLine</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </widget>
       <widget class="QWidget" name="orders_list">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <attribute name="title">
         <string>Orders List</string>
        </attribute>
        <widget class="QTableWidget" name="orders_table_widget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>1222</width>
           <height>817</height>
          </rect>
         </property>
         <property name="gridStyle">
          <enum>Qt::DotLine</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1237</width>
     <height>29</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menu_assembly_listing">
    <property name="title">
     <string>Assemblies/Items</string>
    </property>
    <addaction name="action_new_item"/>
    <addaction name="action_edit_item"/>
    <addaction name="action_edit_conditions"/>
    <addaction name="separator"/>
    <addaction name="action_edit_assembly_tree"/>
    <addaction name="action_save_assembly_list"/>
    <addaction name="separator"/>
    <addaction name="action_update_assemby_tree"/>
   </widget>
   <widget class="QMenu" name="menu_parts">
    <property name="title">
     <string>Parts</string>
    </property>
    <addaction name="action_new_part"/>
    <addaction name="action_edit_part"/>
    <addaction name="action_update_sources"/>
    <addaction name="separator"/>
    <addaction name="action_change_part_number"/>
    <addaction name="action_update_part_list_table"/>
   </widget>
   <widget class="QMenu" name="menu_orders">
    <property name="title">
     <string>Orders</string>
    </property>
    <addaction name="action_new_order"/>
    <addaction name="action_edit_order"/>
    <addaction name="separator"/>
    <addaction name="action_update_order_table"/>
   </widget>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menu_file_recent">
     <property name="title">
      <string>Recent Files</string>
     </property>
     <addaction name="action_recent_file_1"/>
     <addaction name="action_recent_file_2"/>
     <addaction name="action_recent_file_3"/>
     <addaction name="action_recent_file_4"/>
    </widget>
    <addaction name="action_file_open"/>
    <addaction name="action_file_close"/>
    <addaction name="action_file_new"/>
    <addaction name="separator"/>
    <addaction name="menu_file_recent"/>
    <addaction name="separator"/>
    <addaction name="action_file_exit"/>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_assembly_listing"/>
   <addaction name="menu_parts"/>
   <addaction name="menu_orders"/>
   <addaction name="menu_help"/>
  </widget>
  <action name="action_update_assemby_tree">
   <property name="text">
    <string>Update List</string>
   </property>
  </action>
  <action name="action_save_assembly_list">
   <property name="text">
    <string>Save Assembly Listing</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="action_edit_item">
   <property name="text">
    <string>Edit an Item</string>
   </property>
  </action>
  <action name="action_new_item">
   <property name="text">
    <string>Add an Item</string>
   </property>
  </action>
  <action name="ActionItemDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="action_edit_assembly_tree">
   <property name="text">
    <string>Edit Structure</string>
   </property>
  </action>
  <action name="action_update_part_list_table">
   <property name="text">
    <string>Update List</string>
   </property>
  </action>
  <action name="action_edit_part">
   <property name="text">
    <string>Edit a Part</string>
   </property>
  </action>
  <action name="action_new_part">
   <property name="text">
    <string>Add a Part</string>
   </property>
  </action>
  <action name="ActionPartDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="action_change_part_number">
   <property name="text">
    <string>Change Part Number</string>
   </property>
  </action>
  <action name="action_update_order_table">
   <property name="text">
    <string>Update List</string>
   </property>
  </action>
  <action name="action_edit_order">
   <property name="text">
    <string>Edit an Order</string>
   </property>
  </action>
  <action name="action_new_order">
   <property name="text">
    <string>Add an Order</string>
   </property>
  </action>
  <action name="ActionOrderDelete">
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>Delete an Order Completely</string>
   </property>
  </action>
  <action name="action_file_open">
   <property name="text">
    <string>Open Parts File</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="action_file_close">
   <property name="text">
    <string>Close Parts File</string>
   </property>
  </action>
  <action name="action_recent_file_1">
   <property name="text">
    <string>File1</string>
   </property>
  </action>
  <action name="action_recent_file_2">
   <property name="text">
    <string>File2</string>
   </property>
  </action>
  <action name="action_recent_file_3">
   <property name="text">
    <string>File3</string>
   </property>
  </action>
  <action name="action_recent_file_4">
   <property name="text">
    <string>File4</string>
   </property>
  </action>
  <action name="action_file_exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="action_file_new">
   <property name="text">
    <string>New Parts File</string>
   </property>
  </action>
  <action name="action_edit_conditions">
   <property name="text">
    <string>Edit Available Item Conditions</string>
   </property>
  </action>
  <action name="action_update_sources">
   <property name="text">
    <string>Add/Update Part Sources</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
